isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/nutzungsvorgaben/inhalt.adoc

```diff
@@ -391,6 +391,69 @@ Der in `isy.security.roles-claim-name` konfigurierte Wert unterstützt keine Ver
 Es ist sicherzustellen, dass das konfigurierte Roles-Claim auch vom Identity Provider befüllt werden.
 ====
 
+
+[[konfiguration-von-multi-iam-uris]]
+=== Konfiguration von mehreren IAM-URIs
+Zur Konfiguration von mehreren IAM-URIs ist es erforderlich, die Konfigurationsproperties `issuer-uri` und `jwk-set-uri` entsprechend zu definieren und mit besonderem Property-Pfad zu gruppieren.
+
+Der bisherige Property-Pfad
+
+- `*spring*.security.oauth2.resourceserver.jwt` ist durch
+- `*isy*.security.oauth2.resourceserver.jwt.tenants.*<tenant-id>*` zu ersetzen,
+
+wobei die `tenant-id` einzig als Gruppierungskriterium für die beiden Properties `issuer-uri` und `jwk-set-uri` eingesetzt wird.
+Somit kann diese beispielsweise mit fachlich aussagekräftigen Namen, wie der Name der Geschäftsanwendung, des Kommunikationskanals oder der Netzwerkzone, versehen werden.
+
+Die nachstehende Konfigurationstabelle zeigt eine Liste von Property-Pfad-Beispielen, um mehrere IAM-URIs zu definieren.
+Aus Gründen der Übersichtlichkeit wird das Präfix `isy.security.oauth2.resourceserver.jwt` in der unten aufgeführten Tabelle weggelassen.
+
+[[table-property-path-examples-for-multi-tenancy]]
+.Property-Pfad-Beispiele für Multi IAM-URIs
+[cols="3,7",options="header"]
+|===
+|Parameter |Beschreibung
+| tenants.*portal*.issuer-uri
+tenants.*portal*.jwk-set-uri| Property-Set zur Definition einer *externen* und *internen* IAM-URI für den Mandant *Portal*
+| tenants.*sgw-1*.issuer-uri
+tenants.*sgw-1*.jwk-set-uri| Property-Set zur Definition einer *externen* und *internen* IAM-URI für den Mandant *ServiceGateway-1*
+| tenants.*sgw-2*.issuer-uri
+tenants.*sgw-2*.jwk-set-uri| Property-Set zur Definition zweier *interner* IAM-URIs für den Mandant *ServiceGateway-2*
+| tenants.*batch*.issuer-uri
+tenants.*batch*.jwk-set-uri| Property-Set zur Definition zweier *interner* IAM-URIs für den Mandant *Batch*
+| tenants.*ga-A-1*.issuer-uri
+tenants.*ga-A-1*.jwk-set-uri| Erstes Property-Set zur Definition einer *externen* und *internen* IAM-URI für den Mandant *Geschäftsanwendung-A*
+| tenants.*ga-A-2*.issuer-uri
+tenants.*ga-A-2*.jwk-set-uri| Zweites Property-Set zur Definition zweier *interner* IAM-URIs für den Mandant *Geschäftsanwendung-A*
+| tenants.*ga-A-3*.issuer-uri
+tenants.*ga-A-3*.jwk-set-uri| Drittes Property-Set zur weiteren Definition zweier *interner* IAM-URIs für den Mandant *Geschäftsanwendung-A*
+|===
+
+Beispiele für Property-Werte sind in der <<listing-multi-tenancy-configuration-example>> zu finden.
+
+==== Verwendung und Auswertung des Property-Sets
+
+*Property `issuer-uri`*
+
+Das Property `issuer-uri` wird im Normalfall - für die Rolle der _"Authentifizierung startenden Anwendung"_- mit der URI des _Authentifizierungsservers_ des IAM initialisiert.
```

Comment:
Nein, in diesem Fall wird der Konfigurationsparamter "client.provider.<providerId>.issuer-uri" verwendet. Weder über "tenants....issuer-uri", noch über "tenants...jwk-set-uri" wird eine Authentifizierung angestoßen.